tests/unit/pipeline/extractors/test_files.py

```diff
@@ -36,39 +41,39 @@ def fixture_directory():
 
 
 @pytest.fixture
-def unsupported_file(fixture_directory):
+def unsupported_file(fixture_directory: str) -> Path:
     with NamedTemporaryFile(
         "w+", suffix=".unsupported", dir=fixture_directory, delete=False
     ) as temp_file:
         name = temp_file.name
         temp_file.write("hello world")
         temp_file.seek(0)
-    yield Path(name)
+    return Path(name)
```

Comment:
no need to yield, there's no cleanup here.  The context manager `temp_file` is closed by the time the 'yield' gets hit, so to better satisfy the type checkers I flipped these to returns.